testsuite/MDAnalysisTests/analysis/test_dihedrals.py

```diff
@@ -40,7 +78,7 @@ def universe(self):
 
     def test_ramachandran(self, universe):
         rama = Ramachandran(universe.select_atoms("protein")).run()
-        test_rama = np.load(DihedralsArray)
+        test_rama = np.load(RamaArray)
```

Comment:
make it a fixture (see comments below for `JaninArray` – this is the common way to treat "reference data" in pytest